packages/compass-collection/src/components/create-tab/create-tab.jsx

```diff
@@ -17,11 +17,11 @@ class CreateTab extends PureComponent {
    * Create a new tab with the same namespace as the last one.
    */
   createTab = () => {
-    this.props.createNewTab(
-      this.props.activeNamespace,
-      this.props.activeIsReadonly,
-      this.props.activeSourceName
-    );
+    this.props.createNewTab({
```

Comment:
Ah we need to send it here too - looks like this might also be a small bug with not supplying the `sourcePipeline` as well